iroh/src/commands.rs

```diff
@@ -318,6 +321,15 @@ pub enum NodeCommands {
 impl NodeCommands {
     pub async fn run(self, client: RpcClient) -> Result<()> {
         match self {
+            Self::Connections => {
+                let mut connections = client.server_streaming(ConnectionsRequest).await?;
+                println!("Connections:");
+                while let Some(Ok(Ok(ConnectionsResponse { node_info }))) = connections.next().await
```

Comment:
@rklaehn can I get some RPC help?

I don't understand how I ended up with double `Result`s. Did I do something incorrectly, or is expected?

Reply:
This is expected behavior. The first `Result` is around a connection error while reading the response, and the second `Result` is the `RpcResult` that you are sending as `Item` in the stream.